### PR TITLE
Clicker improvements

### PR DESCRIPTION
**Pull request by PavlyukovYegor**
## Description

Clicker improvements for TV:
* Add cursor
* Handle keyboard
* Calculation max and min cursor positions

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/menu/view/main_menu_page_tv.dart [hunk 1]
```diff
@@ -36,7 +36,7 @@ class _MainMenuPageTvState extends State<MainMenuPageTv> {
 
     _playerController = VideoPlayerController.asset(Assets.images.earth)
       ..setLooping(true)
-      ..play()
+      // ..play()
```
Comment (nikitaessen) [inline lib/menu/view/main_menu_page_tv.dart]: check if that's a nesessary change

File: lib/menu/view/main_menu_page_tv.dart [hunk 2]
```diff
@@ -46,7 +46,7 @@ class _MainMenuPageTvState extends State<MainMenuPageTv> {
 
   void _onPlayerStopped() {
     if (!_playerController.value.isPlaying) {
-      _playerController.play();
+      // _playerController.play();
```
Comment (nikitaessen) [inline lib/menu/view/main_menu_page_tv.dart]: check if that's a nesessary change

File: lib/menu/widgets/main_menu_background.dart [hunk 3]
```diff
@@ -37,7 +37,7 @@ class _MainMenuBackgroundState extends State<MainMenuBackground>
 
     _playerController = VideoPlayerController.asset(Assets.images.earth)
       ..setLooping(true)
-      ..play()
+      // ..play()
```
Comment (nikitaessen) [inline lib/menu/widgets/main_menu_background.dart]: check if that's a nesessary change

File: lib/menu/widgets/main_menu_background.dart [hunk 4]
```diff
@@ -75,7 +75,7 @@ class _MainMenuBackgroundState extends State<MainMenuBackground>
 
   void _onPlayerStopped() {
     if (!_playerController.value.isPlaying) {
-      _playerController.play();
+      // _playerController.play();
```
Comment (nikitaessen) [inline lib/menu/widgets/main_menu_background.dart]: check if that's a nesessary change

File: lib/app/view/app.dart [hunk 5]
```diff
@@ -17,14 +17,14 @@ class App extends StatelessWidget {
...
   Widget build(BuildContext context) {
-    SystemChrome.setPreferredOrientations([
-      DeviceOrientation.portraitUp,
-      DeviceOrientation.portraitDown,
-      if (ExtendedPlatform.isTv) ...[
-        DeviceOrientation.landscapeLeft,
-        DeviceOrientation.landscapeRight,
-      ],
-    ]);
+    // SystemChrome.setPreferredOrientations([
```
Comment (nikitaessen) [inline lib/app/view/app.dart]: can you explain why we need to comment this out?

**Review by nikitaessen — COMMENTED** [review]
please address comments I left

File: lib/menu/view/main_menu_page_tv.dart (hunk 1 above)
Comment (PavlyukovYegor) [reply lib/menu/view/main_menu_page_tv.dart]: fixed

File: lib/menu/view/main_menu_page_tv.dart (hunk 2 above)
Comment (PavlyukovYegor) [reply lib/menu/view/main_menu_page_tv.dart]: fixed

File: lib/menu/widgets/main_menu_background.dart (hunk 3 above)
Comment (PavlyukovYegor) [reply lib/menu/widgets/main_menu_background.dart]: fixed

File: lib/menu/widgets/main_menu_background.dart (hunk 4 above)
Comment (PavlyukovYegor) [reply lib/menu/widgets/main_menu_background.dart]: fixed

File: lib/app/view/app.dart (hunk 5 above)
Comment (PavlyukovYegor) [reply lib/app/view/app.dart]: fixed

**Review by nikitaessen — APPROVED** [review]

**Review by DenisovAV — APPROVED** [review]

**Pull request merged**